.github/workflows/main.yml

```diff
@@ -91,6 +92,7 @@ jobs:
 
       - uses: actions/upload-artifact@v2
         with:
+          python-version: 3.9
```

Comment:
This doesn't work. Try `if: ${{ matrix.python-version }} == '3.9'` here (above with)